ui/src/app/base/components/NotificationGroup/NotificationGroup.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from "classnames";
+import pluralize from "pluralize";
+import PropTypes from "prop-types";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { Button, Notification } from "@canonical/react-components";
+import type { Dispatch } from "redux";
+
+import { capitaliseFirst } from "app/utils";
+import { useVisible } from "app/base/hooks";
+import { notification as notificationActions } from "app/base/actions";
+import type { Notification as NotificationType } from "app/store/notification/types";
+
+export const notificationTypes = {
```

Comment:
Can this be an `enum`?

I think the convention would be to have something like:

```typescript
enum NotificationTypes = {
Caution = "caution",
...
}
```

Reply:
Now that I look at this again I've noticed it isn't even used anywhere!